### DISCUSSION TITLE
supabase-storage-bucket-kotlin

### DISCUSSION THREAD

**Discussion by sudophils** [supabase/supabase#9977] [Questions] [answered]
Hi Guys.
Was wondering where i can get access to the **bucket to download or get the download url from a bucket**, just as i could do with the flutter version. Kindly point me to the kotlin variant, thanks

**Comment by GaryAustin1** [comment, marked as answer]
I have not used it and don’t know the status but is this what you are looking for https://github.com/supabase-community/supabase-kt/tree/master/Storage ?

**Reply by sudophils** [reply]
Alright thank you, I found there is a kotlin version for bucket interaction
